requirements.txt

```diff
@@ -4,3 +4,4 @@ ch_util @ git+https://github.com/chime-experiment/ch_util.git
 cora @ git+https://github.com/radiocosmology/cora.git
 driftscan @ git+https://github.com/radiocosmology/driftscan.git
 draco @ git+https://github.com/radiocosmology/draco.git
+ephem
```

Comment:
In theory ephem should be deprecated and removed from all our code. I'd rather not have it sneaking back in. My plan was basically to remove it everywhere, and then if there is anything still using ephem it will break and get fixed to use Skyfield instead. Do you absolutely need it to here to build the docs? If so, can you rework things such that it can be removed from the requirements.

Reply:
`ephem.Sun()` is used by `ch_pipeline.analysis.solar` only. Can I replace that with something? `caput.time.skyfield_wrapper.ephemeris["sun"]`?

Reply:
That's probably fine. I'm pretty sure that the new ephemeris changes have already broken that code anyway.